src/rules.ts

```diff
@@ -397,6 +397,27 @@ export const rules: Rule[] = [
         ),
       ],
   ),
+  new Rule(
+      'Remove Multiple Spaces',
+      'Removes or multiple consecutive spaces. Ignores Spaces at the beginning and ending of the line. ',
```

Comment:
`Removes or multiple consecutive spaces.` - think the grammar is off.
spaces should be lowercase

Reply:
fixed